vmdb/db/migrate/20150501193927_default_provider_verify_ssl.rb

```diff
@@ -0,0 +1,9 @@
+class DefaultProviderVerifySsl < ActiveRecord::Migration
+  def up
+    change_column :providers, :verify_ssl, :integer, :null => false
```

Comment:
Yes, if you need the current set of nils to be removed, there should probably be a data migration.